src/software/jetson_nano/setup_nano.sh

```diff
@@ -1,35 +1,42 @@
 #!/bin/bash
 
+set -ex
 
-# (sorted alphabetically)
 host_software_packages=(
+    redis
     libjpeg8-dev
     libjpeg-dev
-    python3.8       # Python 3
+    python3.8      # Python 3
     python3.8-venv # Virtual Environment
+    python3.8-dev  # Python 3 Headers
     zlib1g-dev
 )
 
 sudo apt-get update
-
 sudo add-apt-repository -y ppa:deadsnakes/ppa
 
-if ! sudo apt-get install "${host_software_packages[@]}" -y ; then
-    echo "##############################################################"
-    echo "Error: Installing utilities and dependencies failed"
-    echo "##############################################################"
```

Comment:
would keeping the error statements ever help in case we need to look at the setup script log?

Reply:
I've replaced that with `set -ex` so the script will stop and error out if any of the commands fail